modules/ghlint/src/main/kotlin/net/twisterrob/ghlint/reporting/TextReporter.kt

```diff
@@ -21,5 +21,6 @@ private fun Finding.render(): String {
 			else -> "${start.line.number}:${start.column.number}-${end.line.number}:${end.column.number}"
 		}
 	}
-	return "${issue.id} at ${location.file.path}:${loc}: ${message}"
+	val hint = "For more information about this rule, run: ghlint --help ${issue.id}"
```

Comment:
Adding the hint message is helpful for users! However, this changes the output format which might break existing tooling that parses the output. Consider making this configurable or documenting the format change.